site/app/libraries/database/txt.sql

```diff
@@ -0,0 +1,28 @@
+SELECT * FROM(
```

Comment:
The content of this `txt.sql` file seems to be the same as the code in `DatabaseQueries.php`. Is this file needed or used somewhere?

Reply:
Mistakenly added from test I was doing. Removed